lib/transforms/push.js

```diff
@@ -31,12 +31,16 @@ function mapFile(githubFile) {
 
 function createJobData(payload, jiraHost) {
   // Store only necessary repository data in the queue
+  const {
+    id, name, full_name, html_url, owner,
+  } = payload.repository;
+
   const repository = {
-    id: payload.repository.id,
-    name: payload.repository.name,
-    full_name: payload.repository.full_name,
-    html_url: payload.repository.html_url,
-    owner: payload.repository.owner,
+    id,
+    name,
+    full_name,
+    html_url,
+    owner,
```

Comment:
cool little trick with this, we can do something like:
```
const repository = {
    id,
    name,
    full_name,
    html_url,
    owner,
} = payload.repository;
```
:)

Reply:
When this change I get `Unexpected use of 'name'  no-restricted-globals` as name is a reserved word. Should I refactor the naming of that variable or leave with the current changes?

Reply:
does it work with `name:name` instead?

Reply:
actually, nvm, apparently you can't do it in one like as it doesn't omit the other properties.  Weird.  Leave it as is :)